emf/src/test/java/com/powsybl/emf/IGMmergeTests.java

```diff
@@ -199,6 +199,7 @@ void testCompareSubnetworksMergeAgainstAssembled() {
             dl.setQ0(0);
         }
         Network assembled = createCGM();
+        //TODO This test fails because in `assembled` the dangling lines have non-zero values for p0 and q0.
```

Comment:
@zamarrenolm 
`IGMMergeTests.testCompareSubnetworksMergeAgainstAssembled` fails with `powsybl-core:6.2.0-RC1`. I think it is because of powsybl-core#2775.
Now, in this test, the dangling lines in `assembled` have non-zero values for `p0` and `q0` (the same values as in `merged`). Thus modifying `merged` before the comparison introduces a difference between the 2 networks.

What is the expected behaviour for `Network.read(...)` with several IGM? Should we get dangling lines with `p0 = q0 = 0.0` or do we want to keep those non-zero values?

Reply:
You are right @olperr1, the change in behaviour is related to powsybl-core#2775.

Since this PR, when we import several IGMs in one call to `Network.read` we expect each IGM to be imported as a subnetwork, keeping all its individual information available, and then combining them in a single network using `Network.merge`. 

We think that is best to keep the original non-zero values. If we do this, each IGM/subnetwork is _correct_ individually, and also the complete network (the equivalent injections from each side of a boundary will "cancel" when we look at the complete network).

So the proposal here is, after upgrading to core 6.2.0-RC1 and later, is to remove the previous lines that reset the values of p0, q0 for the paired dangling lines, leaving the test as:

```java
Network merged = Network.merge("merged",
    Network.read(CgmesConformity1Catalog.microGridBaseCaseBE().dataSource()),
    Network.read(CgmesConformity1Catalog.microGridBaseCaseNL().dataSource()));
Network assembled = createCGM();
compareNetwork(assembled, merged);
```

Reply:
Thank you @zamarrenolm.
I changed the unit test to adopt your recommendations.